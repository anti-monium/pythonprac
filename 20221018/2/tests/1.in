:cat "meow"
:e^ x exp(x)
cat
cat
e^ 3
:len s len(s)
:foo x y z w exp(e)+w-sin(z/y)
len "qqqyaqq"
foo 1 2 3 4
quit "Super{}, class{}"
