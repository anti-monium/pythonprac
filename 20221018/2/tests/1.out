meow
meow
20.085536923187668
7
18.15676725487521
Super5, class10
